my-app/src/redux/game/actions.js

```diff
@@ -0,0 +1,4 @@
+export const actions = {
+  handleClick: 'HANDLECLICK',
```

Comment:
Usually, since these are constants, we usually name these action names in constant case:
```
HANDLE_CLICK: 'HANDLE_CLICK'
```

Reply:
`handleClick` is not a very descriptive action. Maybe `submitPlay` or something alike ?